### ISSUE TITLE
Sign become unreadable when items placed in Small Storage Crates nearby

### ISSUE THREAD

**Issue by YonesBones** [Cazadorsniper/PO3#193]
**Pack Version:**
Pack version is 3.0.38

**Description:**
After setting up signs on Small Storage Crates by Actually Additions as labels then putting anything inside the crates or in the Bonsai pots, the letters become solid squares on the signs. Even if I edit or replace the signs they are still unreadable.

**Steps to reproduce:**
Place Small Storage Crate
Place a couple of signs near the crate
Add Bonsai Pot on top of crates
Add Sapling to Bonsai Pot
**OR**
Add an item into the crate

**Screenshot below:**
![2019-03-23_17 25 23](https://user-images.githubusercontent.com/48863349/54868802-c529ef80-4d90-11e9-8aa4-d01196397d7c.png)

**Latest log file:**
[latest.log](https://github.com/Cazadorsniper/PO3/files/2999586/latest.log)

**Comment by Cazadorsniper** [comment]
Does it happen to other storage like chests?

**Comment by YonesBones** [comment]
> 
> 
> Does it happen to other storage like chests?

So after some testing in creative I discovered that it does. It happens with vanilla Minecraft chests and the Quark ones too. But the issue, I think, is the bonsai pots. I tested just putting random items in all three types of chests with signs on and next to them. Nothing happened, signs were fine. But when I redid the same test with the Hopping Bonsai pots, the signs become unreadable once I placed saplings in there (used vanilla saplings). I also used the Bibliocraft fancy signs too to see what would happen.

**Here are some screenshots**
![2019-03-24_18 43 16](https://user-images.githubusercontent.com/48863349/54883469-3d161980-4e66-11e9-9e2c-5850bab041b4.png)

It changes as I look around
![2019-03-24_18 43 20](https://user-images.githubusercontent.com/48863349/54883476-4e5f2600-4e66-11e9-8b32-02d20d588454.png)

I wanted to see if the signs would change back if I removed the Bonsai pots, they did.
**With Bonsai Pots**
![2019-03-24_18 48 59](https://user-images.githubusercontent.com/48863349/54883488-6040c900-4e66-11e9-96a6-cb864f813fae.png)

**After removing Bonsai pots**
![2019-03-24_18 49 08](https://user-images.githubusercontent.com/48863349/54883518-8cf4e080-4e66-11e9-83ff-5b14881fa44e.png)
